pkg/skaffold/runner/runner.go

```diff
@@ -172,6 +172,21 @@ func (r *SkaffoldRunner) Run(ctx context.Context, out io.Writer, artifacts []*v1
 		return errors.Wrap(err, "deploy step")
 	}
 
+	if !r.opts.Tail {
```

Comment:
@priyawadhwa do you think the duplication could be removed?

Reply:
I thought about moving it into a separate function, but I think four things would have to be passed in, so it would look like:

```golang
func StreamLogsFromDeployedObjects(ctx context.Context, out io.Writer, builds []build.Artifact, artifacts []*v1alpha2.Artifact)
```
and I personally didn't think that was much cleaner, but I don't have a strong opinion so can definitely move it into a function if you prefer that!

Reply:
Let's merge this one any think about how to remove the duplication afterwards!